crypto/modes/gcm128.c

```diff
@@ -413,6 +413,17 @@ void gcm_ghash_rv64i_zbc(u64 Xi[2], const u128 Htable[16],
                          const u8 *inp, size_t len);
 void gcm_ghash_rv64i_zbc__zbkb(u64 Xi[2], const u128 Htable[16],
                                const u8 *inp, size_t len);
+/* Zvbb/Zvbc (vector crypto with vclmul) based routines. */
+void gcm_init_rv64i_zvbb_zvbc(u128 Htable[16], const u64 Xi[2]);
+void gcm_gmult_rv64i_zvbb_zvbc(u64 Xi[2], const u128 Htable[16]);
+void gcm_ghash_rv64i_zvbb_zvbc(u64 Xi[2], const u128 Htable[16],
+                               const u8 *inp, size_t len);
+/* Zvkg (vector crypto with vghmac.vv). */
```

Comment:
vghmac has been renamed to vghsh in RISC-V vector crypto specification.

Reply:
Fixed